src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/IndirectionNode.cs

```diff
@@ -12,19 +12,40 @@ namespace ILCompiler.DependencyAnalysis
     /// </summary>
     public class IndirectionNode : ObjectNode, ISymbolNode
     {
-        private ISymbolNode _indirectedNode;
+        public class IndirectionNodeData
```

Comment:
Made this a Tuple. No need for a new structure.

---
In reply to: [105775146](https://github.com/dotnet/corert/pull/2962#discussion_r105775146) [](ancestors = 105775146)